src/locales/en/common.json

```diff
@@ -1653,7 +1653,8 @@
     "walletConnectCallRequestRejectFailed": "Failed to reject Call Request.",
     "walletConnectRequestRejected": "Wallet Connect Call Request rejected.",
     "walletConnectRequestApproved": "Wallet Connect Call Request approved.",
-    "ensMigrationCannotProceed": "Sorry, ENS migration cannot proceed. Please contact support."
+    "ensMigrationCannotProceed": "Sorry, ENS migration cannot proceed. Please contact support.",
```

Comment:
Do we need "Please contanct support" part, there is a Toast.show option `supportLink: true` which displays interactive support link button

Reply:
This was per Kieran suggestion, @IAmKio wdyt?

Reply:
You know i'm just wondering if this was there at a time when we didn't have the support link (pre-toast rework)? Because yeah it would read "Please contact support. Contact Support" 😂 so i'm for removing it.

Reply:
🚀